components/base/Card.tsx

```diff
@@ -15,39 +15,58 @@ import { ROBOTO, POPPINS } from "../typography";
 import { breakDown } from "../breakpoints";
 
 type Props = {
-  link?: boolean;
+  link?: { href?: string; external?: boolean };
   centerAlign?: boolean;
   icon: string;
   badge?: string;
   topic: string;
   description: string;
 };
 
-function Card({ description, badge, centerAlign, icon, link, topic }: Props) {
-  return (
-    <CardStyle className={centerAlign ? "center" : ""}>
-      <StyledIcon src={icon} alt="" className={centerAlign ? "center" : ""} />
-
-      {badge && (
-        <BadgeStyle>
-          <Badge text={badge} />
-        </BadgeStyle>
-      )}
-
-      {link && (
-        <LinkStyle>
-          <NewTabIcon width={13} />
-        </LinkStyle>
-      )}
-
-      <TopicStyle className={centerAlign ? "center" : ""}>{topic}</TopicStyle>
-
-      <DescriptionStyle className={centerAlign ? "center" : ""}>
-        {description}
-      </DescriptionStyle>
-    </CardStyle>
+const CoreCard = ({
+  description,
+  badge,
+  centerAlign,
+  icon,
+  link,
+  topic,
+}: Props) => (
+  <CardStyle className={centerAlign ? "center" : ""}>
+    <StyledIcon src={icon} alt="" className={centerAlign ? "center" : ""} />
+
+    {badge && (
+      <BadgeStyle>
+        <Badge text={badge} />
+      </BadgeStyle>
+    )}
+
+    {link && link.external && (
+      <LinkIconStyle>
+        <NewTabIcon width={13} />
+      </LinkIconStyle>
+    )}
+
+    <TopicStyle className={centerAlign ? "center" : ""}>{topic}</TopicStyle>
+
+    <DescriptionStyle className={centerAlign ? "center" : ""}>
+      {description}
+    </DescriptionStyle>
+  </CardStyle>
+);
+
+const Card = ({ link, ...rest }: Props) =>
+  link && link.href ? (
+    <Link href={link.href} passHref>
+      <StyledLink
+        target={link.external && "_blank"}
+        rel={link.external && "noopener noreferrer"}
+      >
+        <CoreCard link={link} {...rest} />
+      </StyledLink>
+    </Link>
+  ) : (
+    <CoreCard link={link} {...rest} />
```

Comment:
Adding an `href` to the `link` prop will allow the linking to be handled within the component rather than outside of it, but it is not required. This should also leave the door open should we ever want clicking a card to trigger an event rather than linking somewhere